modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+
+[[changed-plugins-paste]]
+==== Paste
+
+The `paste` functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+Because the functionality was unmaintained, `paste` no longer supports input from Microsoft Word.
+
+There is an open https://github.com/tinymce/tinymce/discussions/7487[request for maintainers] should someone in the community be interested in taking over maintainance of this particular functionality as a separate plugin.
+
+If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
+
+NOTE: the Premium plugin, https://tiny.cloud/docs/plugins/premium/powerpaste/[`PowerPaste`], is available. It provides the capability to accept data from Microsoft Word and Microsoft Excel, and clean-up the received data before pasting it into place.
+
+[[changed-plugins-print]]
+==== Print
+
+`print` functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `print` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[spellchecker]]
+==== Spellchecker
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `spellchecker` plugin has been removed. It is no longer part of {productname} 6.0.
+
+The https://tiny.cloud/tinymce/features/#productivity[Premium plugin], https://tiny.cloud/docs/plugins/premium/tinymcespellchecker/[Spellchecker Pro], which offers equivalent functionality and more, is available.
+
+[[tabfocus]]
+==== TabFocus
+
+The `tabfocus` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+The `tabfocus_elements` option, which relied on the `tabfocus` plugin, no longer functions.
+
+As well, the `tab_focus` option was previously deprecated by `tabfocus_elements`. With this release, the `tab_focus` option has been removed completely.
+
+As per standard web practice, the `tabindex` attribute should be used instead of the `tabfocus` plugin or any of the `tabfocus` plugin’s options.
+
+As per that standard web practice, the `tabindex` attribute is copied from the target element to the iframe.
+
+[[changed-plugins-table-of-contents]]
+==== Table of contents
+
+The `toc` plugin is no longer part of the Core open source editor. _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+NOTE: although there are no end-user facing changes, as part of the general architectural change, the `tableofcontents` Premium plugin has modernised its output. It uses the ES2020 library and targets ES2018. It also uses https://terser.org[terser] to improve performance.
```

Comment:
```suggestion
NOTE: Although there are no end-user facing changes, as part of the general architectural change, the `tableofcontents` Premium plugin has modernised its output. It uses the ES2020 library and targets ES2018. It also uses https://terser.org[terser] to improve performance.
```

Reply:
Similar to my previous comment, the terser change really doesn't have anything to do with performance. It's a minifier so the only perf gain you can get from that is loading speed and we're not going to get anything noticeable from swapping from UglifyJS to Terser. We also shouldn't mention anything about ES2020 library/targets as that's an internal build thing and would likely only confuse customers.
```suggestion
NOTE: Although there are no end-user facing changes, as part of the general architectural change, the `tableofcontents` Premium plugin has modernised its output.
```